modules/network/pre-existing-vpc/main.tf

```diff
@@ -18,10 +18,24 @@
 data "google_compute_network" "vpc" {
   name    = var.network_name
   project = var.project_id
+
+  lifecycle {
+    postcondition {
+      condition     = self.name != null
+      error_message = "The network provided could not be found."
```

Comment:
Is it possible to print the network name and project id as part of the error message using `var.network_name`. This would give the user more immediate information for debugging why the network was not found.

Reply:
Done